### PR TITLE
add @elv1s as CODEOWNERS

### PR DESCRIPTION
**Pull request by dhmlau**
### Description
Welcome @elv1s to be the maintainer of the PostgreSQL connector. 🎉 

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # Each line is a file pattern followed by one or more owners,
 # the last matching pattern has the most precedence.
 
-# Core team members from IBM
-* @jannyHou @b-admike @dhmlau @zbarbuto
-
 # Alumni members
 # @kjdelisle @loay @ssh24 @virkt25
+
+# Core team members from IBM
```
Comment (bajtos) [inline CODEOWNERS]: `elv1s` is not from IBM, is he?

```suggestion
# Core team members
```

Is there any particular reasons why you are moving the list of active maintainers at the bottom, after alumni members? I believe our other repos use the convention where active members are listed first.

**Comment by bajtos** [comment]
Welcome @elv1s on board! 🎉

File: CODEOWNERS (hunk 1 above)
Comment (dhmlau) [reply CODEOWNERS]: You're right. I'll remove the `from IBM`. 

For the order, when i made the changes, I put the active ones at the bottom, seems easier to see.  But I'm good either way, just want to be consistent.  There are other connector repos that should be updated as well. :) 

Active maintainers at the bottom, using `#` as prefix:
https://github.com/strongloop/loopback-connector-mssql/blob/master/CODEOWNERS
https://github.com/strongloop/loopback-connector/blob/master/CODEOWNERS

Alumni at the bottom and using `_` as prefix:
https://github.com/strongloop/loopback/blob/master/CODEOWNERS

Comment (bajtos) [reply CODEOWNERS]: I can see arguments for both (active at the bottom, alumni at the bottom). No need to bike shed, feel free to pick the order that makes most sense to you.

Out of scope of this PR: should we start building list of alumni maintainers in https://github.com/strongloop/loopback-next/blob/master/CODEOWNERS too?

**Pull request merged**